test-cypress/stubs/monitoring-stub.js

```diff
@@ -0,0 +1,11 @@
+const DELETE_QUERY_ALIAS = 'deleteSparqlQuery';
+export class MonitoringStub {
+
+    static interceptDeleteSparqlQuery(repositoryId) {
+        cy.intercept('DELETE', `rest/monitor/repository/${repositoryId}/query?*`).as(DELETE_QUERY_ALIAS);
+    }
+
+    static verifyADeleteQueryExecuted() {
```

Comment:
Don't do this! Stub classes are intended only for stubbing. Do this check in the test!

Reply:
Done